bench/generation/perplexity.py

```diff
@@ -64,7 +275,7 @@ def perplexity(
     stride: int = 512,
 ):
     dtype = torch.float32 if device.type == "cpu" else torch.float16
-    tokenizer = AutoTokenizer.from_pretrained(model_id)
+    tokenizer = AutoTokenizer.from_pretrained(model_id, use_fast=False)
```

Comment:
I set it to `False` since the slow tokenizer is better at processing long string. I only takes a few seconds to process the dataset.